samples/dashboards/demo/project-management/demo.html

```diff
@@ -0,0 +1,16 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+<script src="https://code.highcharts.com/gantt/modules/gantt.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container"></div>
+
+<p class="highcharts-description">
+    This demo shows how to use the Highcharts dashboards to create a project
+    management dashboard. <br>
```

Comment:
Prefer using paragraphs, not single breaks. https://github.com/highcharts/highcharts/blob/master/repo-guidelines.md#1-avoid-line-breaks-without-paragraphs